cassdegrees/templates/student_edit.html

```diff
@@ -81,7 +81,7 @@ <h3>Courses</h3>
                     </div>
 
 
-                    <ul class="pagination"></ul>
+                    <div class="pagination align-center"></div>
```

Comment:
This a div on purpose?

Reply:
Yes, as ul added indent that messed up with the formatting in the student plan editor which forced some buttons to the next line. I tried removing it by defining styling for ul tags in the css file but it didn't work. Changing to div did the trick.